src/Compilers/Core/Portable/Compilation/Compilation.cs

```
@@ -1370,13 +1371,15 @@ internal bool FilterAndAppendDiagnostics(DiagnosticBag accumulator, IEnumerable<
                     continue;
                 }
 
+
                 var filtered = Options.FilterDiagnostic(d);
                 if (filtered == null ||
                     (!reportSuppressedDiagnostics && filtered.IsSuppressed))
                 {
                     continue;
                 }
-                else if (filtered.Severity == DiagnosticSeverity.Error)
+                else if (filtered.Severity == DiagnosticSeverity.Error &&
+                         !filtered.IsSuppressed)
```

>!filtered.IsSuppressed) [](start = 25, length = 23)

can errors be suppressed? I thought only warnings could.